code/modules/RCD/engie.dm

```diff
@@ -47,3 +47,58 @@
 /obj/item/weapon/rcd_ammo/attackby(var/obj/O, mob/user)
 	if(is_type_in_list(O, list(/obj/item/device/rcd/matter/engineering,  /obj/item/device/rcd/matter/rsf)) || (istype(O, /obj/item/device/material_synth) && !istype(O, /obj/item/device/material_synth/robot)))
 		return O.attackby(src, user)
+
+/obj/item/device/rcd/matter/engineering/mech
+	matter = 30
+
+/obj/item/device/rcd/matter/engineering/mech/use_energy(var/amount, var/mob/user)
```

Comment:
If you don't need the matter usage code, you shouldn't inherit from matter RCDs and just make an `/obj/item/device/rcd/engineering/mech` directly.